interactions/api/models/guild.py

```diff
@@ -768,6 +768,27 @@ async def create_channel(
 
         return Channel(**res, _client=self._client)
 
+    async def clone_channel(self, channel_id: int) -> Channel:
+        """
+        Clones a channel of the guild.
+
+        :param channel_id: The id of the channel to clone
+        :type channel_id: int
+        :return: The created (cloned) channel
+        :rtype: Channel
+        """
+        if not self._client:
+            raise AttributeError("HTTPClient not found!")
+        res = await self._client.get_channel(channel_id=channel_id)
+
+        res["permission_overwrites"] = [Overwrite(**_) for _ in res["permission_overwrites"]]
+        [
+            res.pop(attr, None)
+            for attr in {"flags", "last_pin_timestamp", "guild_id", "id", "last_message_id"}
+        ]
```

Comment:
Might as well make it just a for loop instead of a list comprehension if black formats it as 4 lines
Also sorted the attributes alphabetically so it looks nicer
```suggestion
        for attr in {"flags", "guild_id", "id", "last_message_id", "last_pin_timestamp"}:
            res.pop(attr, None)
```